crates/types/src/net/log_server.rs

```diff
@@ -298,6 +316,29 @@ pub struct Stored {
     pub header: LogServerResponseHeader,
 }
 
+impl IntoBilrostDto for Stored {
+    // Since Stored message is just the header, we can implement
+    // IntoDto directly into LogServerResponseHeader type
+    type Target = dto::LogServerResponseHeader;
```

Comment:
This makes it impossible to add other fields to `Stored` message on the wire without breaking wire compatibility, right?

If yes, I'm not sure if I'd want that.